### DISCUSSION TITLE
hcloud load balancer services port change

### DISCUSSION THREAD

**Discussion by cuminpepper** [mysticaltech/terraform-hcloud-kube-hetzner#406] [Q&A] [answered]
Hi, I created a new cluster with deafult `kube.tf.example`. How to modify the services in hcloud? The default 80 port points to 32263 and I want to change it, e.g., 30001. Thank you.

**Comment by mysticaltech** [comment, marked as answer]
@cuminpepper You should not change those, they are selected automatically by the Hetzner CCM.
